omod-2.0/src/test/java/org/openmrs/module/unrelatedtest/rest/resource/UnrelatedGenericChildResource.java

```diff
@@ -25,33 +26,27 @@
 @Resource(name = RestConstants.VERSION_1 + "/unrelated", supportedClass = UnrelatedGenericChild.class, supportedOpenmrsVersions = { "1.9.* - 9.*" })
 public class UnrelatedGenericChildResource extends GenericChildResource {
 	
-	public static boolean getGETCalled = false;
+	public static boolean getRepresentationDescription = false;
 	
-	public static boolean getCREATECalled = false;
-	
-	public static boolean getUPDATECalled = false;
-	
-	/*******************************
-	 * TEST METHOD IMPLEMENTATIONS * These methods are the ones we want to test against. There
-	 * implementaion is unimportant, they just set flags so we can assert the methods were called
-	 * correctly by the reflector.
-	 */
+	public static boolean getCreatableProperties = false;
 	
+	public static boolean getUpdatableProperties = false;
+
 	@Override
-	public Model getGETModel(Representation rep) {
-		getGETCalled = true;
-		return super.getGETModel(rep);
+	public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
```

Comment:
Did you intentionally change these methods?

Reply:
yes i did, because we are getting rid of those methods so  i had to replace them with methods used for creating the rest api